dotnetv3/EC2/Actions/SsmWrapper.cs

```diff
@@ -25,21 +25,19 @@ public async Task<List<Parameter>> GetParametersByPath(string path)
     {
         var parameters = new List<Parameter>();
         var request = new GetParametersByPathRequest { Path = path };
-        var response = await _amazonSSM.GetParametersByPathAsync(request);
 
-        // Make sure we get the whole list.
-        do
+        // Make sure we get the whole list with a paginator.
```

Comment:
My inner Liz wants this to read:
```suggestion
        // Get the whole list with a paginator.
```

Reply:
Good catch! Added.